packages/fr-config-push/src/helpers/config-process.js

```diff
@@ -38,3 +45,4 @@ function removeProperty(obj, propertyName) {
 
 module.exports.replaceEnvSpecificValues = replaceEnvSpecificValues;
 module.exports.removeProperty = removeProperty;
+module.exports.unescapePlaceholders = unescapePlaceholders;
```

Comment:
Instead of exporting the unescapePlaceholders function, call it after replacing the non-escaped placeholders. then the function call stays the same.